source/FallingBlock.hx

```diff
@@ -0,0 +1,67 @@
+package;
+
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import haxe.Timer;
+
+/**
+ * A block that falls after being stepped on
+ * @author TR Nale
+ */
+class FallingBlock extends Block
```

Comment:
If I recall correctly, rising blocks are also a thing in Mario, we might want to make this class such that it can handle both rising and falling blocks.